pkgs/games/prismlauncher/default.nix

```diff
@@ -11,6 +11,7 @@
 , qtbase
 , quazip
 , extra-cmake-modules
+, testers
```

Comment:
don't want to block on this, but I guess we don't need to include testers if we're not using `testVersion` mentioned earlier... otherwise all good